### PR TITLE
fix: Turning on test now that mirror node issue #7036 is fixed.

### LINKED ISSUE #721: Enable checks that were disabled while mirrornode issue #7036 was open.
### Problem

Tests were disabled during issue.

### Solution

Re-enable existing test.

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by ebadiere**
Re-enable tests

Fixes #721 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: test/solidity/block/BlockInfo.js [hunk 1]
```diff
@@ -18,9 +18,10 @@
  *
  */
 const { expect } = require('chai');
-const { ethers } = require('hardhat');
+const { BigNumber, ethers } = require('hardhat');
 const Constants = require('../../constants');
-
+// const { BigNumber } = require('ethers');
```
Comment (quiet-node) [inline test/solidity/block/BlockInfo.js:23]: nit: remove comment

File: test/solidity/block/BlockInfo.js [hunk 2]
```diff
@@ -59,34 +60,34 @@ describe('@solidityequiv1 BlockInfo Test Suite', function () {
     expect(coinbase).to.equal('0x0000000000000000000000000000000000000062');
   });
 
-  // Turn off until mirror node issue is resolved: https://github.com/hashgraph/hedera-mirror-node/issues/7036
   it('should get the current block prevrandao using block.prevrandao', async function () {
+    let prevrandao;
     try {
-      await blockInfo.getBlockPrevrando();
+      prevrandao = await blockInfo.getBlockPrevrando();
+      console.log('prevrandao', prevrandao);
```
Comment (quiet-node) [inline test/solidity/block/BlockInfo.js:67]: nit: does this log help in any way? If not I'd suggest clean it up.

**Review by quiet-node — APPROVED** [review]
LGTM. Couple of nits

File: test/solidity/block/BlockInfo.js (hunk 1 above)
Comment (ebadiere) [reply test/solidity/block/BlockInfo.js:23]: Done.

File: test/solidity/block/BlockInfo.js (hunk 2 above)
Comment (ebadiere) [reply test/solidity/block/BlockInfo.js]: No, sorry, cleaned up.

**Review by quiet-node — APPROVED** [review]
LGTM

**Pull request merged**